docs/components/importdoc.vue

```diff
@@ -178,27 +175,15 @@ export default {
         return {
           directive: this.directiveAttr(d),
           namedExport: d,
-          importPath: this.directivePath(d)
+          importPath: `bootstrap-vue`
```

Comment:
We can just use single quotes here.

Reply:
Ah true... so used to typing backticks on something else...

may make this a constant and reference.

Reply:
Yeah